stories/Ads.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { Story, Meta } from "@storybook/react";
+import { Ad } from "@playpickup/core";
+import { AdZoneProps } from "../packages/core/src";
+
+export default {
+  title: "Core/Ad",
+  component: Ad,
+  argTypes: {
+    height: {
+      defaultValue: "125",
+      description: "Height of Ad",
+    },
+    width: {
+      defaultValue: "600",
+      description: "Width of Ad",
+    },
+    adProps: {
```

Comment:
Let's separate the props out here to make them individually editable in Storybook